app/assets/stylesheets/nav.scss

```diff
@@ -133,10 +133,10 @@
 
 .nav-lozenge {
   white-space: nowrap;
-  background: #232B49;
-  background-image: linear-gradient(180deg, #252E50 0%, #374268 99%);
+  background: $dark-blue;
```

Comment:
Have changed this so we use $dark-blue as the base and taking shades of it, rather than adding more colours to the palette